pkg/cmd/roachtest/test_impl_test.go

```diff
@@ -138,6 +138,22 @@ func Test_failuresMatchingError(t *testing.T) {
 			},
 			want: true,
 		},
+		{
+			name: "an error contains the expected error, as part of a multi-error",
```

Comment:
Nit: "...expected error _type_...". Maybe it's just me, but I was staring at `targetError{errors.New("expected-error")})` and `refError: targetError{errors.New("some error")}` long enough to warrant the extra qualifier.

Reply:
Yes, sorry about the confusion. Updated.